src/InternPool.zig

```diff
@@ -1098,7 +1102,8 @@ const Local = struct {
         @sizeOf(u64) => List(struct { u64 }),
         else => @compileError("unsupported host"),
     };
-    const Strings = List(struct { u8 });
+    const LargeStrings = List(struct { offset: u32, len: u32 });
```

Comment:
You shouldn't need to store a length, it can be derived by storing an extra entry in an offset list and subtracting two adjacent offsets.

Reply:
Without this change this branch is 0.3-0.4% slower than master.  With this change is 0.3-0.4% faster than master, where the difference between using small strings and only using large strings is <0.1%, therefore I have decided that the extra complexity is not worth such a small speedup and we will just use this large string strategy for all strings.

Reply:
All versions are an extra 1-3MB over master.